docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Autopilot Pattern NFS server implementation
+
+nfsserver:
+    image: autopilotpattern/nfsserver
+    privileged: true
```

Comment:
This is pretty bad juju on Docker-on-Linux. Can we make this more fine-grained with `--cap-add` or `--cap-drop` or `--device`?